README.md

```diff
@@ -15,7 +15,7 @@ An interactive CLI tool for building and testing [XLS-102 WASM](https://github.c
 
 ## Installation
 
-```bash
```

Comment:
Why are all these not run in the Markdown test? They should work successfully.

Reply:
> I suppose this does test the actual comment is not malformed, so maybe it's valuable

Yeah, this is my thought.

> e.g., line 77 doesn't work ATM

That's actually something that should get caught by the test, because it's wrong. It should be `craft build notary  -O aggressive`